### PR TITLE
feat: use sandpack with md in docs

### LINKED ISSUE #5040: Use vitepress-sandbox to render editor demo in docs
It is recommended to use [vitepress-sandbox-plugin](https://vitepress-sandbox.js-bridge.com/get-started/introduction.html) to render demos in MD.

[install docs](https://vitepress-sandbox.js-bridge.com/get-started/install.html)

For example, it is now used by [blocknotejs](https://www.blocknotejs.org/docs/image-toolbar).

Preview screen:

image

Code in md, write:

image

It supports dark automatic

image

### PR DESCRIPTION
**Pull request by jerrywu001**
fix #5040 

1. 升级vitepress到最新版本
2. 集成vitepress-sandbox

image

image

### CODE REVIEW

**Comment by jerrywu001** [comment]
我的账号应该没有权限部署文档

File: packages/docs/components/code-sandbox.vue [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      ...props, // do not forget it
+      coderHeight: Number(props.coderHeight),
+      previewHeight: Number(props.previewHeight),
+      showLineNumbers: true,
+    }"
+    :custom-setup="{
+      ...props, // do not forget it
+      deps: {
+        yjs: 'latest',
+        '@toeverything/theme': 'latest',
```
Comment (doodlewind) [inline packages/docs/components/code-sandbox.vue:18]: Just curious if we can use the `nightly` version here?

**Review by doodlewind — APPROVED** [review]
Thanks for the feat! This greatly improves the usability of our getting started docs!

File: packages/docs/components/code-sandbox.vue (hunk 1 above)
Comment (jerrywu001) [reply packages/docs/components/code-sandbox.vue:18]: ok, you can update it.

**Review by doodlewind — COMMENTED** [review]
@jerrywu001 could you please update the `CLA.md` together in this PR? Thanks!

**Comment by jerrywu001** [comment]
> @jerrywu001 could you please update the `CLA.md` together in this PR (or using a standalone one)? Thanks!

ok, wait a momment

File: packages/docs/components/code-sandbox.vue [hunk 2]
```diff
@@ -15,7 +15,7 @@
       ...props, // do not forget it
       deps: {
         yjs: 'latest',
-        '@toeverything/theme': 'latest',
```
Comment (doodlewind) [inline packages/docs/components/code-sandbox.vue]: I think this package doesn't release in nightly channel.

**Pull request merged**